agenta-cli/sdk.py

```diff
@@ -0,0 +1,652 @@
+from typing import Any, Dict, Optional
+
+from pydantic import BaseModel
+
+
+class Reference(BaseModel):
+    # IDENTIFIER
+    id: Optional[str] = None
+    slug: Optional[str] = None
+    # VERSION
+    version: Optional[str] = None
+
+
+class Application(BaseModel):
+    # SCOPE
+    project_id: str
+    # IDENTIFIER
+    id: str
+    slug: str
+    # HEADER
+    name: Optional[str] = None
+    description: Optional[str] = None
+
+
+class Service(BaseModel):
+    # INTERNALS
+    url: str
+
+
+class Configuration(BaseModel):
```

Comment:
In the current state of things, configuration is an API route, yet it is not an entity/table, but just a jsonb in the variant.
Should we keep having this for the app structure 2, or just remove the entity all together?